cmd/oras/tag/tag.go

```diff
@@ -26,14 +32,15 @@ type tagOptions struct {
 	option.Common
 	option.Remote
 
-	srcRef    string
-	targetRef string
+	defaultTagNOptions oras.TagNOptions
```

Comment:
Can we just call it for better readability?
```golang
concurrency int64
```

Reply:
done